scripts/ansible/roles/golang_base/tasks/centos-7.yml

```diff
@@ -0,0 +1,12 @@
+- name: Install Go repo
+  yum:
+    name:
+      - epel-release
+
+- name: Install Go
+  become: true
+  yum:
+    name:
+      - golang  # 1.13.11 in July 2020
```

Comment:
These versions are likely to not change anymore right?

Reply:
On CentOS 7 this role obtains golang from EPEL, so the version has changed. Today it would get [golang-1.15.2-1.el7](http://mirrors.ukfast.co.uk/sites/dl.fedoraproject.org/pub/epel/7/x86_64/Packages/g/golang-1.15.2-1.el7.x86_64.rpm)

Reply:
So is there no problem in having the go version not fixed?

Reply:
I am asking because we have specified (actually currently some minimum) go requirements for the version and just wanting to make sure this may not resulting in installing some unsupported version

Reply:
I haven't seen it cause a problem, but I haven't tested every golang version released since I first wrote this. I was relying on the *or higher* aspect of "Software: [Go](https://golang.org/doc/install) version >= 1.13.X" including, 1.14.x, 1.15.x etc. However as I write this I'm realising that the README might mean gloang >= 1.13.x, *and* golang , 1.14.0.

If you prefer, I can disable the CentOS cases until/if there's better test coverage.

Reply:
Nevermind, thanks